pages/docs/couple-your-code/couple-your-code-initializing-coupling-data.md

```diff
@@ -5,19 +5,17 @@ keywords: api, adapter, initialization, coupling scheme, restart
 summary: "As default values, preCICE assumes that all coupling variables are zero initially. For fluid-structure interaction, for example, this means that the structure is in its reference state. Sometimes, you want to change this behavior – for instance, you may want to restart your simulation."
 ---
 
-For initializing coupling data, you can add the following **optional** method:
+{% version 3.0.0 %}
+`initializeData()` only exists for preCICE versions < 3.0. For versions >= 3.0, data initialization is directly handled in `initialize()`.
+{% endversion %}
+
+By default preCICE assumes that all coupling variables are zero initially. If you want to provide non-zero initial values, you can write data before calling `initialize()`. This data will then be used as initial data. The preCICE action interface that was introduced in [Step 6](couple-your-code-implicit-coupling) provides the following action to check whether initial data has to be written by a participant:
 
 ```cpp
-void initializeData();
+const std::string& constants::actionWriteInitialData()
```

Comment:
Needs updates. "action interface" is gone.

Reply:
Done cdd2062